src/main/java/org/gridsuite/study/server/service/SupervisionService.java

```diff
@@ -197,5 +197,9 @@ private Map<UUID, String> formatSubreportMap(String subReporterKey, List<Network
             node -> node.getId() + "@" + subReporterKey)
         );
     }
+
+    public void invalidateAllNodeBuilds(UUID studyUuid) {
```

Comment:
Remove this method

Reply:
Kept and removed from `StudyService` as discussed